Update readme style (#194)
OfficeDev/PnP-OfficeAddins: Samples/office-contextual-tabs/README.md

# Create custom contextual tabs on the ribbon

This sample accomplishes the following tasks using Office ribbon APIs.

- Creates a custom contextual tab named **Table Data**.
- Creates a table in Excel. When the focus is inside the table, the custom tab is displayed.
- When the focus is outside the table, the custom tab is hidden.

> ![Screen shot that shows when a table in Excel has the focus, a custom contextual tab named Table Data is shown on the ribbon](pnp-add-contextual-tabs-to-your-add-in.png)

Reviewer: ```suggestion
![Screenshot that shows when a table in Excel has the focus, a custom contextual tab named Table Data is shown on the ribbon.](pnp-add-contextual-tabs-to-your-add-in.png)
```

## Applies to

- Excel on Windows
- PowerPoint on Windows

## Prerequisites

- Microsoft 365

## Run the sample

> Run this sample in Excel in a browser. The add-in web files are served from this repo on GitHub.

Author: good catch! Should be fixed now.

# Create custom contextual tabs on the ribbon

This sample accomplishes the following tasks using Office ribbon APIs.

- Creates a custom contextual tab named **Table Data**.
- Creates a table in Excel. When the focus is inside the table, the custom tab is displayed.
- When the focus is outside the table, the custom tab is hidden.

![Screenshot that shows when a table in Excel has the focus, a custom contextual tab named Table Data is shown on the ribbon.](pnp-add-contextual-tabs-to-your-add-in.png)

## Applies to

- Excel on Windows

> - Excel on web

Reviewer: ```suggestion
- Excel on the web
```

## Key parts of this sample

> This sample inserts a table of fictitious sales data for Contoso. The data is pulled from one of two mock data sources; a mock Excel file, or a mock SQL database. The user can select which data source to use either in the task pane, or in the contextual tab.

Reviewer: ```suggestion
This sample inserts a table of fictitious sales data for Contoso. The data is pulled from one of two mock data sources: a mock Excel file, or a mock SQL database. The user can select which data source to use either in the task pane, or in the contextual tab.
```

## Questions and comments

> We'd love to get your feedback about this sample. Please send your feedback to us in the Issues section of this repository. Questions about developing Office Add-ins should be posted to [Microsoft Q&A](https://docs.microsoft.com/en-us/answers/topics/office-js-dev.html) using the office-js-dev tag.

Reviewer: ```suggestion
We'd love to get your feedback about this sample. Please send your feedback to us in the Issues section of this repository. Questions about developing Office Add-ins should be posted to [Microsoft Q&A](https://docs.microsoft.com/answers/topics/office-js-dev.html) using the office-js-dev tag.
```

## Additional resources

- [Create custom contextual tabs in Office Add-ins](https://docs.microsoft.com/office/dev/add-ins/design/contextual-tabs)

Demonstration video:
[![YouTube video showing the contextual tab sample code and how it works](https://img.youtube.com/vi/9tLfm4boQIo/0.jpg)](https://www.youtube.com/watch?v=9tLfm4boQIo)

## Solution

> Solution | Author(s)

Reviewer: I think screen reader says "author" "s"
```suggestion
Solution | Authors
```
